### PR TITLE
single subscribe

### PR DESCRIPTION
**Pull request by vikiival**
This feature is needed for #2292 

The base is that when you component need to flexibly subscribe to single subscription with the flexible arguments

- :zap: added singleSubscribe Mixin
- :bug: linter fails if you have newer node than 16.13.2

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### CODE REVIEW

**Comment by chandradot99** [comment]
@vikiival I Mixins can be used in Vue component, but in our case of #2292, we are subscribing either from plugin or from Vuex Store, so I think we can use this Mixin there.

File: utils/mixins/singleSubscribeMixin.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+declare type Unsubscribe = () => void
+
+/*
+ * refer to https://stackoverflow.com/questions/51873087/unable-to-use-mixins-in-vue-with-typescript
+ * usage import Component, { mixins } from 'vue-class-component';
+ * class ExtendedClass extends mixins(SubscribeMixin) {
+ */
+@Component
+export default class SingleSubscribeMixin extends Vue {
+  private sub: Unsubscribe = () => void 0
```
Comment (roiLeo) [inline utils/mixins/singleSubscribeMixin.ts:12]: `void 0`?

**Review by roiLeo — COMMENTED** [review]
I don't really understand why and what this new mixins is for, but the code looks good to me.

**Comment by vikiival** [comment]
Like the problem with plugin is that you can't unsubscribe

File: utils/mixins/singleSubscribeMixin.ts (hunk 1 above)
Comment (vikiival) [reply utils/mixins/singleSubscribeMixin.ts:12]: > why and what this new mixin is for

The use-case is that @chandradot99 has to subscribe for the user's balance.
We also need to correctly handle when the user logs in under different address

**Pull request merged**